menambahkan halaman index

brambrc opened the pull request:
# Login Homepage

Halaman homepage sederhana dengan tampilan login yang dibuat menggunakan HTML dan CSS biasa.

## 📋 Deskripsi

Project ini berisi halaman login yang bersih dan modern sebagai homepage. Dibuat dengan teknologi web dasar (HTML + CSS) tanpa menggunakan framework eksternal.

## 🚀 Fitur

- **Desain Modern**: Tampilan yang bersih dengan gradient background
- **Form Login**: Input field untuk username dan password
- **Responsive Design**: Kompatibel dengan desktop dan mobile
- **Smooth Animations**: Transisi halus dan hover effects
- **User Experience**: Interface yang user-friendly
- **Bahasa Indonesia**: Semua teks menggunakan bahasa Indonesia

## 📁 Struktur File

```
├── index.html      # Halaman utama dengan form login
├── style.css       # Stylesheet untuk seluruh styling
├── test.html       # File test (existing)
└── README.md       # Dokumentasi project
```

## 🎨 Teknologi yang Digunakan

- **HTML5**: Struktur halaman dan semantic markup
- **CSS3**: Styling lengkap dengan:
  - Flexbox untuk layout
  - CSS Grid untuk positioning
  - Gradient backgrounds
  - Transitions dan animations
  - Media queries untuk responsive design

## 💻 Cara Menjalankan

1. Clone atau download repository ini
2. Buka file `index.html` di browser web
3. Halaman login akan langsung ditampilkan

```bash
# Jika menggunakan live server
open index.html
```

## 🎯 Komponen Halaman

### Form Login
- **Username Field**: Input untuk username/email
- **Password Field**: Input untuk password (tersembunyi)
- **Remember Me**: Checkbox untuk mengingat login
- **Login Button**: Tombol untuk submit form

### Additional Links
- **Lupa Password**: Link untuk reset password
- **Daftar Akun Baru**: Link untuk registrasi

## 📱 Responsive Design

Halaman ini sudah dioptimasi untuk berbagai ukuran layar:
- **Desktop**: Layout penuh dengan maksimal width 400px
- **Mobile**: Padding dan font size yang disesuaikan
- **Tablet**: Tampilan yang fleksibel

## 🎨 Color Scheme

- **Primary Gradient**: #667eea → #764ba2
- **Background**: Linear gradient purple-blue
- **Text**: #333 (dark gray)
- **Borders**: #e1e1e1 (light gray)
- **Focus State**: #667eea (blue)

## 📝 To-Do / Future Enhancements

- [ ] Integrasi dengan backend authentication
- [ ] Validasi form menggunakan JavaScript
- [ ] Dark mode toggle
- [ ] Social login options
- [ ] Password strength indicator
- [ ] Loading states untuk button

## 🤝 Contributing

1. Fork project ini
2. Buat feature branch (`git checkout -b feature/AmazingFeature`)
3. Commit perubahan (`git commit -m 'Add some AmazingFeature'`)
4. Push ke branch (`git push origin feature/AmazingFeature`)
5. Buat Pull Request

## 📄 License

Project ini bersifat open source dan bebas digunakan untuk keperluan pembelajaran.

---

**Dibuat dengan ❤️ menggunakan HTML & CSS**

Commit:
menambahkan halaman index

dyolin commented on index.html:
```diff
@@ -0,0 +1,42 @@
+<body>
+    <div class="login-container">
+        <div class="login-box">
+            <h2>Selamat Datang</h2>
```
ganti Selamat Datang menjadi Welcome

brambrc replied:
ok, otw betulin

Commit:
Add login homepage with responsive design

- Create index.html as main homepage with login form
- Update style.css with comprehensive styling for login page
- Add modern gradient background and smooth animations
- Implement responsive design for mobile compatibility
- Include README.md with complete project documentation
- Support Indonesian language throughout the interface
